dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/AbstractOracleConnectorTest.java

```diff
@@ -44,4 +50,52 @@ public void buildProperties_success() {
     assertEquals(EXAMPLE_PASSWORD, actual.getProperty("password"));
     assertEquals(EXAMPLE_USER, actual.getProperty("user"));
   }
+
+  @Test
+  public void buildUrl_providedBoth_throwsException() {
+    when(arguments.getOracleServicename()).thenReturn("ORCLPDB");
+    when(arguments.getOracleSID()).thenReturn("ORCLPDB1");
+    // Act
+    ThrowingRunnable runnable = () -> AbstractOracleConnector.buildUrl(arguments);
+    // Assert
+    assertThrows(MetadataDumperUsageException.class, runnable);
```

Comment:
I would recommend keeping the `assertThrows` call in the Act section and inlining the runnable.

See example usage:
https://github.com/google/dwh-migration-tools/blob/531ca76b2b1cb2cb383fa4b5923a7b44e1eb34db/dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/QueryLogTableNamesResolverTest.java#L142-L144

Similarly for other tests, where `assertThrows` is used.

Reply:
Done, I moved `assertThrows` inside `// Act` to match convention

Reply:
I would still slightly suggest to keep the Act step with a single assignment only as is already done in other tests, following the pattern:
```
E e = assertThrows(E.class, () -> ...);
```

Reply:
Done